src/e2ee/worker/e2ee.worker.ts

```diff
@@ -119,11 +119,11 @@ onmessage = (ev) => {
 async function handleRatchetRequest(data: RatchetRequestMessage['data']) {
   if (useSharedKey) {
     const keyHandler = getSharedKeyHandler();
-    await keyHandler.ratchetKey(data.keyIndex);
+    await keyHandler.ratchetKey(data.keyIndex, undefined, data.extracable);
```

Comment:
also only a small nit, just to make the behaviour slightly more obvious here

```suggestion
    await keyHandler.ratchetKey(data.keyIndex, true, data.extracable);
```